lib/helper/request-store.js

```diff
@@ -60,13 +60,33 @@ const putArtifact = async (buildId, token, fileName, file) => {
     };
 
     try {
-        return request(options);
+        return await request(options);
     } catch (err) {
         throw new Error(err.response.body.toString());
     }
 };
 
+const deleteZip = async (buildId, token) => {
```

Comment:
[small comment] this function name is inconsistent with `getZipArtifact`

Reply:
thx.
I fixed it.